target/linux/ramips/dts/mt7621_xiaoyu_xy-c5.dts

```diff
@@ -93,10 +93,6 @@
 	status = "okay";
 };
 
-&ethernet {
-	mtd-mac-address = <&factory 0xe000>;
```

Comment:
And now you get a random MAC address?

Reply:
thank you for notice about the mac address, I test it and here is the result 
before delete: I get something static, not bootloader setting,not factory address.
after delete: I get the mac address in my bootloader setting,(is factory mac if I not change it).

Reply:
`mtd-mac-address = <&factory 0x4>;`
label MAC address

Reply:
Thanks.

Is there a second address on flash for WAN?
If not, is it calculated or kept the same as LAN?

Reply:
No second address on flash.
`wan_mac=lan_mac + 1` in factory firmware

Reply:
it is true .

Reply:
@shNanChen It's working